src/ProjectTemplates/Web.ProjectTemplates/content/ComponentsWebAssembly-CSharp/Client/Shared/MainLayout.NoAuth.razor

```diff
@@ -7,7 +7,7 @@
 
     <main>
         <div class="top-row px-4">
-            <a href="http://blazor.net" target="_blank" class="ml-md-auto">About</a>
+            <a href="https://docs.microsoft.com/aspnet/" target="_blank">About</a>
```

Comment:
Why is this changing?

Reply:
It was randomly inconsistent with the `MainLayout.Auth.razor` file. I think the link I've put here is the correct one out of the two.

Also I've moved these changes to a separate PR now: https://github.com/dotnet/aspnetcore/pull/35296